crates/pop-parachains/src/utils/helpers.rs

```diff
@@ -24,6 +24,35 @@ pub(crate) fn sanitize(target: &Path) -> Result<(), Error> {
 	Ok(())
 }
 
+pub fn is_initial_endowment_valid(initial_endowment: &str) -> bool {
+	match initial_endowment.parse::<u128>() {
+		Ok(_) => return true,
+		Err(_error) => match left_shift(initial_endowment) {
+			Ok(_) => return true,
+			Err(_error) => return false,
+		},
+	}
+}
+fn left_shift(initial_endowment: &str) -> Result<u64, Error> {
```

Comment:
Rename, it is parsing and validating

Something like `is_valid_bitwise_left_shift`

Reply:
Added a comment too as Bruno suggested.